### ISSUE TITLE
Specify a browser version range?

### ISSUE THREAD

**Issue by ben-eb** [ai/browserslist#26]
For example:

```js
var legacyIE = browserslist('ie 5.5 through 8');
//=> ['ie 5.5', 'ie 6', 'ie 7', 'ie 8']
```

**Comment by ai** [comment]
Maybe `ie 7-9` syntax is more better, especially for non-English speakers? :)

**Comment by ai** [comment]
Released in 0.5

**Comment by ben-eb** [comment]
Awesome! :+1:
